selfdrive/car/mazda/values.py

```diff
@@ -44,11 +61,18 @@ class MazdaCarInfo(CarInfo):
   CAR.MAZDA6: MazdaCarInfo("Mazda 6 2017-20"),
   CAR.CX9_2021: MazdaCarInfo("Mazda CX-9 2021-22", video_link="https://youtu.be/dA3duO4a0O4"),
   CAR.CX5_2022: MazdaCarInfo("Mazda CX-5 2022-23"),
+  CAR.MAZDA3_2019: MazdaCarInfo("Mazda 3 2019-23"),
+  CAR.CX_30: MazdaCarInfo("Mazda CX-30 2019-23"),
+  CAR.CX_50: MazdaCarInfo("Mazda CX-50 2022-23"),
+  CAR.CX_60: MazdaCarInfo("Mazda CX-60 unreleased"),
+  CAR.CX_70: MazdaCarInfo("Mazda CX-70 unreleased"),
+  CAR.CX_80: MazdaCarInfo("Mazda CX-80 unreleased"),
+  CAR.CX_90: MazdaCarInfo("Mazda CX-90 unreleased"),
 }
 
 
 class LKAS_LIMITS:
-  STEER_THRESHOLD = 15
+  STEER_THRESHOLD = 1500
```

Comment:
Same

Reply:
I will separate the THRESHOLD's based on GEN1, GEN2. 1500 is a good threshold for GEN2